source/collaborate/send-messages.rst

```diff
@@ -73,6 +73,10 @@ Edit messages
 
   2. Select **Edit** to edit your own messages. Editing a message won't trigger new :doc:`@mention notifications </collaborate/mention-people>`, or :doc:`desktop notifications </preferences/manage-your-notifications>`.
 
+  .. tip::
+
+    From Mattermost v10.5, using a web browser or the Mattermost Desktop app, you can also change or remove message attachments when editing your sent messages.
```

Comment:
@harshilsharma63 - Is the ability to edit/remove message attachments limited to webapp/desktop app only in v10.5?

Reply:
Yes, it is limited to webapp and desktop.